i18n/en/docusaurus-plugin-content-docs/current/appendix.md

```diff
@@ -132,7 +132,7 @@ Parameter “periods” is the interest payment periods.
 
 ### Strategy  
 
-Strategy refers to smart contracts that interface with third-party agreements. BOC Vault has multiple policies, and the policy is one-to-one with external third-party agreements.
+Strategy refers to smart contracts that interface with third-party agreements. BOC Vault has multiple policies, and the policy is one-to-one with external third-party protocols.
```

Comment:
@Francisco-Rua  check "policy/ies" everywehere -- mostly likely should be "strategy/ies"